### ISSUE TITLE
error: results is not an array

### ISSUE THREAD

**Issue by incandescentman** [dtaht/Gnugol#7]
Now when I try to use gnugol, I get:
`error: results is not an array`
Any ideas?

**Comment by dtaht** [comment]
There was some sort of API change in libjannson since 1.3 or 2.0 that is
now breaking things. (it's at like 4.0 now) I have not have time to look
at it.

**Comment by dtaht** [comment]
No, the google API has been obsoleted. I will switch to duckduckgo, probly.

**Comment by Moanrisy** [comment]
any follow up from here?
stackoverflow show the same error

**Comment by dtaht** [comment]
I gave up. duckduckgo's results were so limited as to remove gnugol from being a useful tool.

**Comment by bzg** [comment]
Hello from the past!

**Comment by dtaht** [comment]
Well, I'd like to find a search engine to interface gnugol with. *I* need it, it really helped to be able to use voice syntehsis in particular to cope with my partial blindness, and being about to translate urls on the fly and keep a local cache of my results, really helped with my writing.
